CreditManagement/forms.py

```diff
@@ -40,6 +58,16 @@ class Meta(TransactionForm.Meta):
             'target_user': 'User',
         }
 
+    def clean(self):
+        cleaned_data = super().clean()
+
+        # Do not allow associations to make evaporating money transactons
+        # (not restircted on database level, but it doesn't make sense to order it)
+        if not cleaned_data.get('target_user') and not cleaned_data.get('target_association'):
+            raise ValidationError("Select a target to transfer the money to.")
+
+        return cleaned_data
```

Comment:
Nevermind, I now get that that is done to enable the `UserDonationForm`. It's still better I think though to restrict it generally and make exceptions for e.g. the `UserDonationForm` instead of other way around (but just leave it like this)